interfaces/inc/c_pal/threadpool.h

```diff
@@ -21,10 +21,12 @@ extern "C" {
 #endif
 
 typedef struct THREADPOOL_TAG THREADPOOL;
-typedef struct TIMER_INSTANCE_TAG* TIMER_INSTANCE_HANDLE;
+typedef struct TIMER_TAG TIMER;
```

Comment:
Can we have this close to the THANDLE_TYPE_DECLARE, so that it is not intermingled with the work item stuff?

Reply:
done